.gitignore

```diff
@@ -15,3 +15,6 @@ __pycache__
 # System files
 *~
 .vscode
+
+# Directory of static files for backend server deployment
+dj_hetmech/static
```

Comment:
Ideally I'd like to put this higher up in the gitignore, like before python. Generally I try to put the more generic ignores at the bottom, since they're less related to the actual project.